tasks/main.yml

```diff
@@ -44,37 +51,37 @@
 
 - name: apply admin user role
   command: |
-    {{ bin_dir }}/kubectl apply -f {{ kube_config_dir }}/admin-user.yml
+    {{ kubectl_bin_dir }}/kubectl apply -f {{ kube_config_dir }}/admin-user.yml
 
 - name: get admin-user token
   shell: |
-    echo $({{ bin_dir }}/kubectl -n kube-system describe secret $({{ bin_dir }}/kubectl -n kube-system get secrets | grep admin-user | cut -f1 -d ' ') | grep -E '^token' | cut -f2 -d':' | tr -d '\t\n\r ')
+    echo $({{ kubectl_bin_dir }}/kubectl -n kube-system describe secret $({{ kubectl_bin_dir }}/kubectl -n kube-system get secrets | grep admin-user | cut -f1 -d ' ') | grep -E '^token' | cut -f2 -d':' | tr -d '\t\n\r ')
   register: admin_user_token
 
 - name: create config file
   shell: |
-    {{ bin_dir }}/kubectl config --kubeconfig=config set-cluster internal --server=https://10.233.0.1:443 --insecure-skip-tls-verify &&
-    {{ bin_dir }}/kubectl config --kubeconfig=config set-credentials admin-user --token="{{ admin_user_token.stdout_lines[0] }}" &&
-    {{ bin_dir }}/kubectl config --kubeconfig=config set-context internal --cluster=internal --namespace=default --user=admin-user &&
-    {{ bin_dir }}/kubectl config --kubeconfig=config set-context external --cluster=internal --namespace=default --user=admin-user &&
-    {{ bin_dir }}/kubectl config --kubeconfig=config use-context internal
+    {{ kubectl_bin_dir }}/kubectl config --kubeconfig=config set-cluster internal --server=https://{{ server_ip }}:443 --insecure-skip-tls-verify &&
+    {{ kubectl_bin_dir }}/kubectl config --kubeconfig=config set-credentials admin-user --token="{{ admin_user_token.stdout_lines[0] }}" &&
+    {{ kubectl_bin_dir }}/kubectl config --kubeconfig=config set-context internal --cluster=internal --namespace=default --user=admin-user &&
+    {{ kubectl_bin_dir }}/kubectl config --kubeconfig=config set-context external --cluster=internal --namespace=default --user=admin-user &&
+    {{ kubectl_bin_dir }}/kubectl config --kubeconfig=config use-context internal
 
 - name: get notebook pod name
   shell: |
-    {{ bin_dir }}/kubectl get pods | grep tf-app | cut -d' ' -f1
+    {{ kubectl_bin_dir }}/kubectl get pods | grep tf-app | cut -d' ' -f1
```

Comment:
No need to change, but since IIRC the output of kubectl is YAML, you can use Jinja's `from_yaml` filter to parse the output.

Reply:
Ahh, didn't know you could use jinja for parsing output too, thought it was only for parsing input but of course, it would work either way!